src/main/java/restfulbooking/LocalDateGsonAdapter.java

```diff
@@ -0,0 +1,22 @@
+package restfulbooking;
+
+import com.google.gson.*;
+import com.google.gson.stream.JsonReader;
+import com.google.gson.stream.JsonWriter;
+
+import java.io.IOException;
+import java.time.LocalDate;
+
+public class LocalDateGsonAdapter extends TypeAdapter<LocalDate>
+	{
+	@Override
+	public void write(final JsonWriter jsonWriter, final LocalDate localDate ) throws IOException
+	{
+		jsonWriter.value(localDate.toString());
+	}
+
+	@Override
+	public LocalDate read(final JsonReader jsonReader ) throws IOException {
+		return LocalDate.parse(jsonReader.nextString());
+	}
+}
```

Comment:
This is just for converting date formats.  Copied from example on stackoverflow at:
https://stackoverflow.com/questions/39192945/serialize-java-8-localdate-as-yyyy-mm-dd-with-gson